packages/roosterjs-editor-dom/lib/table/forEachSelectedCell.ts

```diff
@@ -0,0 +1,21 @@
+import VTable from './VTable';
+import { VCell } from 'roosterjs-editor-types';
+
+/**
+ * Executes an action to all the cells within the selection range.
+ * @param callback action to apply on each selected cell
+ * @returns the amount of cells modified
+ */
+export default function forEachSelectedCell(vTable: VTable, callback: (cell: VCell) => void): void {
```

Comment:
Where is this function used? If it only used in CopyPastePlugin but nowhere else, you can put it inside that file and no need to export. Let's try to avoid exporting unnecessary function.